components/eventing-controller/pkg/env/nats_config_test.go

```diff
@@ -16,38 +16,102 @@ func Test_GetNatsConfig(t *testing.T) {
 	idleConnTimeout := time.Second * 40
 
 	envs := map[string]string{
-		"NATS_URL":                "NATS_URL",
-		"EVENT_TYPE_PREFIX":       "EVENT_TYPE_PREFIX",
-		"MAX_IDLE_CONNS":          fmt.Sprintf("%d", maxIdleConns),
-		"MAX_CONNS_PER_HOST":      fmt.Sprintf("%d", maxConnsPerHost),
-		"MAX_IDLE_CONNS_PER_HOST": fmt.Sprintf("%d", maxIdleConnsPerHost),
-		"IDLE_CONN_TIMEOUT":       fmt.Sprintf("%v", idleConnTimeout),
+		"NATS_URL":                 "NATS_URL",
+		"JS_STREAM_NAME":           "kyma",
+		"JS_STREAM_SUBJECT_PREFIX": "kyma",
+		"EVENT_TYPE_PREFIX":        "EVENT_TYPE_PREFIX",
+		"MAX_IDLE_CONNS":           fmt.Sprintf("%d", maxIdleConns),
+		"MAX_CONNS_PER_HOST":       fmt.Sprintf("%d", maxConnsPerHost),
+		"MAX_IDLE_CONNS_PER_HOST":  fmt.Sprintf("%d", maxIdleConnsPerHost),
+		"IDLE_CONN_TIMEOUT":        fmt.Sprintf("%v", idleConnTimeout),
 	}
 
-	g := NewGomegaWithT(t)
 	defer func() {
 		for k := range envs {
-			err := os.Unsetenv(k)
-			g.Expect(err).ShouldNot(HaveOccurred())
+			require.NoError(t, os.Unsetenv(k))
 		}
 	}()
 
 	for k, v := range envs {
-		err := os.Setenv(k, v)
-		g.Expect(err).ShouldNot(HaveOccurred())
+		require.NoError(t, os.Setenv(k, v))
 	}
 
 	maxReconnects, reconnectWait := 1, time.Second
 	config := GetNatsConfig(maxReconnects, reconnectWait)
 
-	g.Expect(config.MaxReconnects).To(Equal(maxReconnects))
-	g.Expect(config.ReconnectWait).To(Equal(reconnectWait))
+	require.Equal(t, config.MaxReconnects, maxReconnects)
+	require.Equal(t, config.ReconnectWait, reconnectWait)
 
-	g.Expect(config.URL).To(Equal(envs["NATS_URL"]))
-	g.Expect(config.EventTypePrefix).To(Equal(envs["EVENT_TYPE_PREFIX"]))
+	require.Equal(t, config.URL, envs["NATS_URL"])
+	require.Equal(t, config.EventTypePrefix, envs["EVENT_TYPE_PREFIX"])
 
-	g.Expect(config.MaxIdleConns).To(Equal(maxIdleConns))
-	g.Expect(config.MaxConnsPerHost).To(Equal(maxConnsPerHost))
-	g.Expect(config.MaxIdleConnsPerHost).To(Equal(maxIdleConnsPerHost))
-	g.Expect(config.IdleConnTimeout).To(Equal(idleConnTimeout))
+	require.Equal(t, config.MaxIdleConns, maxIdleConns)
+	require.Equal(t, config.MaxConnsPerHost, maxConnsPerHost)
+	require.Equal(t, config.MaxIdleConnsPerHost, maxIdleConnsPerHost)
+	require.Equal(t, config.IdleConnTimeout, idleConnTimeout)
+
+	require.Equal(t, config.JSStreamName, envs["JS_STREAM_NAME"])
+	require.Equal(t, config.JSStreamSubjectPrefix, "EVENTTYPEPREFIX")
```

Comment:
(nit) Maybe a comment here would help readers understand why the `EVENTTYPEPREFIX` value is expected.